packages/web/src/scss/components/Dropdown/index.html

```diff
@@ -1,30 +1,297 @@
+        <span>Mauris nunc, elementum enim in lacinia vitae quam placerat sem, euismod accumsan</span>
+      </a>
+      <a href="#" class="d-flex">
+        <svg width="24" height="24" aria-hidden="true" class="mr-400">
+          <use xlink:href="/icons/svg/sprite.svg#help" />
+        </svg>
+        <span>Donec dui, nunc dui vel varius libero molestie nibh nunc</span>
+      </a>
+    </div>
+  </div>
+</section>
+
+<section class="docs-Section">
+
+  <h2 class="docs-Heading">Longer content with auto width feature</h2>
+
+  <div class="DropdownWrapper">
+    <button
+      data-toggle="dropdown"
+      data-target="#DropdownDemo4"
+      data-autoWidth="true"
+      class="Button Button--primary Button--medium"
+      aria-expanded="false"
+      aria-controls="#DropdownDemo4"
+    >
+      Finibus quis imperdiet, semper imperdiet aliquam
+    </button>
+    <div class="Dropdown Dropdown--top Dropdown--left" id="DropdownDemo4">
+      <a href="#" class="d-flex mb-400">
+        <svg width="24" height="24" aria-hidden="true" class="mr-400">
+          <use xlink:href="/icons/svg/sprite.svg#info" />
+        </svg>
+        <span>Bibendum aliquam, fusce integer sit amet congue non nulla aliquet enim</span>
+      </a>
+      <a href="#" class="d-flex mb-400">
+        <svg width="24" height="24" aria-hidden="true" class="mr-400">
+          <use xlink:href="/icons/svg/sprite.svg#link" />
+        </svg>
+        <span>Nulla condimentum, purus commodo cursus non nulla rhoncus</span>
+      </a>
+      <a href="#" class="d-flex mb-400">
+        <svg width="24" height="24" aria-hidden="true" class="mr-400">
+          <use xlink:href="/icons/svg/sprite.svg#profile" />
+        </svg>
+        <span>Mauris nunc, elementum enim in lacinia vitae quam placerat sem, euismod accumsan</span>
+      </a>
+      <a href="#" class="d-flex">
+        <svg width="24" height="24" aria-hidden="true" class="mr-400">
+          <use xlink:href="/icons/svg/sprite.svg#help" />
+        </svg>
+        <span>Donec dui, nunc dui vel varius libero molestie nibh nunc</span>
+      </a>
+    </div>
+  </div>
+</section>
+
+<section class="docs-Section">
+
+  <h2 class="docs-Heading">Longer content with auto width and responsive feature</h2>
+
+  <div class="DropdownWrapper">
+    <button
+      data-toggle="dropdown"
+      data-target="#DropdownDemo5"
+      data-autoWidth="true"
+      class="Button Button--primary Button--medium"
+      aria-expanded="false"
+      aria-controls="#DropdownDemo5"
+      data-breakpoint="tablet"
+    >
+      Finibus quis imperdiet, semper imperdiet aliquam
+    </button>
+    <div class="Dropdown Dropdown--top Dropdown--left" id="DropdownDemo5">
+      <a href="#" class="d-flex mb-400">
+        <svg width="24" height="24" aria-hidden="true" class="mr-400">
+          <use xlink:href="/icons/svg/sprite.svg#info" />
+        </svg>
+        <span>Bibendum aliquam, fusce integer sit amet congue non nulla aliquet enim</span>
```

Comment:
Simple and short items as in the examples above would make this example clearer, I believe.